src/tests/log/sample_log_config.yaml

```diff
@@ -0,0 +1,21 @@
+appenders:
+
+  file:
+    kind: rolling_file
+    path: test_log.log
+    encoder:
+      pattern: "{d(%Y-%m-%d %H:%M:%S)} | {({l}):5.5} | {f}:{L} — {m}{n}"
+    policy:
+      trigger:
+        kind: size
+        limit: 10 kb
```

Comment:
Are you sure that is a right default?

Reply:
Just used to test correct generation of multiple log files